multinode-demo/fullnode.sh

```diff
@@ -268,6 +269,9 @@ while [[ -n $1 ]]; do
     elif [[ $1 = --no-airdrop ]]; then
       airdrops_enabled=0
       shift
+    elif [[ $1 = --recreate-ledger ]]; then
```

Comment:
@pgarg66 - I'm not too happy about the name `--recreate-ledger`, do you have an alternative suggestion perhaps or does that name seem ok to you?

Reply:
How about `reset ledger`?